src/server/index.js

```diff
@@ -37,12 +45,37 @@ app.get('/api/businesses', function(req, res) {
   });
 });
 
-app.use(express.static(path.resolve(__dirname, '../../dist')));
-
 app.get('/login', (req, res) => {
   res.sendFile(path.resolve(__dirname, '../../dist/index.html'));
 });
 
+app.post('/api/login', (req, res) => {
+  login.validation(req, (status) => {
+    if (status === 0) {
+      return res.status(400).
+        json({error: 'content-type should be application/json'});
+    } else if (status === 1){
+      return res.status(400).json({error: 'usename and password required'});
+    }
+  });
+  login.createTokenForExistingUser(req.body,
+    (status) => {
+      if (status === 2) {
+        let Token = jwt.sign({username: req.body.username}, 'epam jsa agate');
```

Comment:
you can use const instead of let and variable **MUST** follow **lowerCamelCase**
`const token`